isso/db/__init__.py

```diff
@@ -121,3 +121,39 @@ def first(rv):
 
                 con.execute('PRAGMA user_version = 3')
                 logger.info("%i rows changed", con.total_changes)
+
+        # "text" field in "comments" became NOT NULL
+        if self.version == 3:
+            with sqlite3.connect(self.path) as con:
+                con.execute("BEGIN TRANSACTION")
+                con.execute("UPDATE comments SET text = '' WHERE text IS NULL")
+                logger.info("%i rows changed", con.total_changes)
+
+                # create new table with NOT NULL constraint for "text" field
+                con.execute('CREATE TABLE IF NOT EXISTS comments_new ('
```

Comment:
This must be kept in sync with [db/comments.py](https://github.com/isso-comments/isso/blob/27bdc72a5e2f4e6ae0a9fd506225774ae4ed711f/isso/db/comments.py). Could you add a note there and also here cautioning authors of this pitfall?
Or even better, refactor a bit so that this executes `db.comments.__init__()` on `comments_new`?